easybuild/easyconfigs/s/SciPy-bundle/SciPy-bundle-2020.02-foss-2020a-Python-3.8.2.eb

```diff
@@ -21,9 +21,9 @@ exts_default_options = {'source_urls': [PYPI_SOURCE]}
 
 # order is important!
 exts_list = [
-    ('numpy', '1.18.1', {
+    ('numpy', '1.18.2', {
```

Comment:
@akesandgren This warrants also changing the `SciPy-bundle` version to `2020.03` (same goes for #10112)

Reply:
Done in #10118